src/items/implementations.md

```diff
@@ -1,32 +1,31 @@
 # Implementations
 
-An _implementation_ is an item that associates items with an *implementing type*.
+An _implementation_ is an item that associates items with an _implementing type_.
+Implementations are defined with the keyword `impl` and contain functions
+that belong to an instance of the type that is being implemented or to the
+type statically.
 
-There are two types of implementations: inherent implementations and [trait]
-implementations.
+There are two types of implementations:
 
-Implementations are defined with the keyword `impl`.
+- inherent implementations
+- [trait] implementations
 
 ## Inherent Implementations
 
 An inherent implementation is defined as the sequence of the `impl` keyword,
 generic type declarations, a path to a nominal type, a where clause, and a
 bracketed set of associable items.
 
-The nominal type is called the *implementing type* and the associable items are
-the *associated items* to the implementing type.
-
-Inherent implementations associate the associated items to the implementing
-type.
+The nominal type is called the _implementing type_ and the associable items are
+the _associated items_ to the implementing type.
 
+Inherent implementations associate the contained items to the implementing type.
 The associated item has a path of a path to the implementing type followed by
-the associate item's path component.
-
-Inherent implementations cannot contain associated type aliases.
+the associate item's path component. Inherent implementations cannot contain
+associated type aliases.
 
-A type can have multiple inherent implementations.
-
-The implementing type must be defined within the same crate.
+A type can also have multiple inherent implementations. Although the
```

Comment:
The "Although" used to join here feels very artificial.

Reply:
Yes, what if I use "but" instead of "although"

Reply:
The "but" implies an exception on the previous statement. An "and" would do well if it cannot read well as two sentences.

Reply:
@Havvy  Does this sound better: "An implementing type must be defined within the same crate as the original type definition."